boot/zephyr/main.c

```diff
@@ -40,6 +40,10 @@ const struct boot_uart_funcs boot_funcs = {
 };
 #endif
 
+#ifdef CONFIG_BOOT_WAIT_FOR_DFU
```

Comment:
I would always use *_USB_DFU or *_usb_dfu if it about USB DFU class, otherwise it is not unique.

Reply:
Done.